builtin/logical/pki/acme_state.go

```diff
@@ -129,6 +131,7 @@ func (a *acmeState) CreateAccount(ac *acmeContext, c *jwsCtx, contact []string,
 		Contact:              contact,
 		TermsOfServiceAgreed: termsOfServiceAgreed,
 		Jwk:                  c.Jwk,
+		Status:               acme.StatusValid,
```

Comment:
I think we'll want this as an internal typed constant I think, rather than a generic string.

Reply:
Or rather, the above should probably be clarified to relate to accounts:

```
type ACMEAccountStatusType string
```

and then be updated above. lol.